src/Atc.Rest.Client/EndpointResponse.cs

```diff
@@ -1,10 +1,12 @@
 using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
 using System.Net;
 
 namespace Atc.Rest.Client
 {
     public class EndpointResponse
     {
+        [SuppressMessage("Design", "CA1062:Validate arguments of public methods", Justification = "OK.")]
         public EndpointResponse(EndpointResponse response)
             : this(response.IsSuccess, response.StatusCode, response.Content, response.ContentObject, response.Headers)
```

Comment:
```suggestion
        public EndpointResponse(EndpointResponse response)
            : this(
                  response?.IsSuccess ?? throw new System.ArgumentNullException(nameof(response)),
                  response.StatusCode,
                  response.Content,
                  response.ContentObject,
                  response.Headers)
```
This is an alternative to suppressing the warning about possible null value of the response parameter. It is not pretty to look at, but since `:this` is executed before the body of the ctor that calls it, we can do the null check inline like so.